spec/README.md

```diff
@@ -16,7 +16,8 @@
    1. [Data Model Errors](errors.md#data-model-errors)
    1. [Resolution Errors](errors.md#resolution-errors)
    1. [Message Function Errors](errors.md#message-function-errors)
-1. [Default Function Registry](registry.md)
+1. [Default Function Registry](registry/default.md)
+1. [`u:` Unicode Registry](registry/unicode.md)
```

Comment:
My design document is calling this the "Unicode Reserved Namespace" and "Unicode Reserved Namespace Registry", since, technically, all of the registries belong to Unicode.